Makefile

```diff
@@ -1,12 +1,8 @@
 default: test
 
 test: env
-	.env/bin/pytest -v
-
-env: .env/.up-to-date
-
-.env/.up-to-date: setup.py Makefile
-	python -m venv .env
-	.env/bin/pip install -e ".[testing]"
-	touch $@
+	uv run --extra testing pytest tests
 
+env:
+	uv venv
+	uv pip install -e ".[testing]"
```

Comment:
Won't this use Python outside of the venv?

Reply:
No, UV use the virtualenv, even if you didn't activate it...